great_expectations/exceptions/__init__.py

```diff
@@ -1 +1,79 @@
-from .exceptions import *
+from .exceptions import (
```

Comment:
For consideration at a later time: this and the next file are going to be a pain to maintain (and maybe surprising for folks adding exceptions/expectations). Using magic should be a last resort, but maybe it's warranted here. Do we want to consider using some cleverness with the `inspect` module to dynamically infer these? That'd also make life easier if we wanted to start exposing `__all__` in these.

Reply:
I will look into this for followup work.

Reply:
totally non-urgent!